lavamoat/browserify/beta/policy.json

```diff
@@ -380,7 +380,7 @@
         "@ethersproject/abi>@ethersproject/bytes": true,
         "@ethersproject/abi>@ethersproject/logger": true,
         "@ethersproject/abi>@ethersproject/properties": true,
-        "ganache>secp256k1>elliptic": true
+        "@metamask/ppom-validator>elliptic": true
```

Comment:
Not sure what this change mean...we're removing ganache and adding ppom-validator? Or is this an auto-generated file.

Reply:
So when we install a new dependency yarn reconfigures where everything is stored at in node_modules based on common resolution paths. What this implies is that both secp256k1 which ganache uses, and metamask/ppom-validator require elliptic, and that  `@ethersproject/signing-key` also requires elliptic and its going to be requesting the package which is being stored in the @metamask/ppom-validator folder in node_modules now instead of the ganache/secp256k1 folder. At least this is my understanding. @weizman @legobeat or @naugtur can explain better but it doesn't mean we removed anything just that it got shifted around in how its resolving.